jiant/tasks/tasks.py

```diff
@@ -2946,6 +3005,66 @@ class SpanPredictionTask(Task):
 
     n_classes = 2
 
+    def update_metrics(self, out, batch):
+        batch_size = len(out["logits"]["span_start"])
```

Comment:
Decided to do it this way since get_output_attirubte necessitates cuda_device, which we don't pass into here.